src/sklearn_evaluation/plot/regression.py

```diff
@@ -0,0 +1,75 @@
+"""
+Plotting functions for regression plots.
+"""
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+from sklearn_evaluation.telemetry import SKLearnEvaluationLogger
+from sklearn.linear_model import LinearRegression
+
+
+@SKLearnEvaluationLogger.log(feature='plot')
+def residual(y_true, y_pred, ax=None):
```

Comment:
can you add a docstring and document the parameters? Look at the other plots as guide

also, please add an inline example. (check out the Examples section in the other plots)

Reply:
Thank you! Will do that